src/test/java/uk/gov/companieshouse/pscdataapi/service/CompanyPscServiceTest.java

```diff
@@ -207,37 +211,76 @@ void insertNewCreatedWhenCreatedCallToMongoFails() {
     @DisplayName("When company number & notification id is provided, delete PSC")
     void testDeletePSC() {
         when(repository.getPscByCompanyNumberAndId(COMPANY_NUMBER, NOTIFICATION_ID)).thenReturn(Optional.ofNullable(pscDocument));
-        service.deletePsc(COMPANY_NUMBER, NOTIFICATION_ID, "");
+        service.deletePsc(new PscDeleteRequest(COMPANY_NUMBER, NOTIFICATION_ID, "", INDIVIDUAL_KIND, DELTA_AT));
 
         verify(repository, times(1)).getPscByCompanyNumberAndId(COMPANY_NUMBER, NOTIFICATION_ID);
         verify(repository, times(1)).delete(pscDocument);
-        verify(chsKafkaApiService).invokeChsKafkaApiWithDeleteEvent(any(), any(), any(), any(), any());
+        verify(chsKafkaApiService).invokeChsKafkaApiWithDeleteEvent(any(), any());
     }
 
     @Test
-    @DisplayName("When company number is null throw ResourceNotFound Exception")
+    @DisplayName("When company number is null throw Bad Request Exception")
     void testDeletePSCThrowsResourceNotFoundException() {
-        when(repository.getPscByCompanyNumberAndId("", NOTIFICATION_ID)).thenReturn(Optional.empty());
+        when(repository.getPscByCompanyNumberAndId("", NOTIFICATION_ID)).thenThrow(BadRequestException.class);
 
-        assertThrows(ResourceNotFoundException.class, () -> service.deletePsc("", NOTIFICATION_ID, ""));
+        assertThrows(BadRequestException.class, () -> service.deletePsc(new PscDeleteRequest("", NOTIFICATION_ID, "", INDIVIDUAL_KIND, DELTA_AT)));
 
         verify(repository, times(1)).getPscByCompanyNumberAndId("", NOTIFICATION_ID);
         verify(repository, never()).delete(any());
-        verify(chsKafkaApiService, never()).invokeChsKafkaApiWithDeleteEvent(any(), any(), any(), any(), any());
+        verify(chsKafkaApiService, never()).invokeChsKafkaApiWithDeleteEvent(any(), any());
     }
 
     @Test
-    @DisplayName("When company number and id is null throw ResourceNotFound Exception")
+    @DisplayName("When company number and id is null throw BadRequestException")
     void testDeletePSCThrowsNotFoundExceptionWhenCompanyNumberAndNotificationIdIsNull() {
```

Comment:
Same here

Reply:
yep, I changed it as part of the above comment - pushing now